.identity/.gitignore

```diff
@@ -0,0 +1,48 @@
+# Local .terraform directories
```

Comment:
I'd remove `.gitignore` from there and I'd add these rules in the main `.gitignore`